src/cuttlefish_util.erl

```diff
@@ -103,12 +103,9 @@ ceiling(X) ->
 
 %% Levenshtein code by Adam Lindberg, Fredrik Svensson via
 %% http://www.trapexit.org/String_similar_to_(Levenshtein)
-%%
-%%------------------------------------------------------------------------------
-%% @spec levenshtein(StringA :: string(), StringB :: string()) -> integer()
+
 %% @doc Calculates the Levenshtein distance between two strings
-%% @end
-%%------------------------------------------------------------------------------
+-spec levenshtein(string(), string()) -> non_neg_integer().
```

Comment:
Unrelated to this change: we now have jaro distance in the standard library. Could we eventually take advantage of that instead? I believe Elixir uses jaro distance for its typo recommendations

https://github.com/erlang/otp/pull/7879